src/libsystemd/sd-journal/journal-file.h

```diff
@@ -123,8 +123,9 @@ typedef struct JournalFile {
 } JournalFile;
 
 typedef enum JournalFileFlags {
-        JOURNAL_COMPRESS = 1 << 0,
-        JOURNAL_SEAL     = 1 << 1,
+        JOURNAL_COMPRESS   = 1 << 0,
+        JOURNAL_SEAL       = 1 << 1,
+        JOURNAL_MULTI_BOOT = 1 << 2,
```

Comment:
maybe add a brief comment here, as what this is for, not's not obvious I think to most readers